### ISSUE TITLE
CodePipeline build step fails if app's Dockerfile is not in root

### ISSUE THREAD

**Issue by efekarakus** [aws/amazon-ecs-cli-v2#893]
### Description

I have a workspace structured as:
```
$ tree
.
├── ecs-project
│   ├── buildspec.yml
│   ├── fe
│   │   └── manifest.yml
│   └──  pipeline.yml
└── fe
    ├── Dockerfile
    ├── go.mod
    ├── go.sum
    └── main.go
```
`pipeline init` and `pipeline update` creates the pipeline successfully, however the `build` step in the pipeline fails:

```

[Container] 2020/05/04 20:51:05 Running command for app in $apps; do
  for docker_dir in $(cat $CODEBUILD_SRC_DIR/ecs-project/$app/manifest.yml \| ruby -ryaml -rjson -e 'puts JSON.pretty_generate(YAML.load(ARGF))' \| jq '.image.build' \| sed 's/"//g'); do
  cd $CODEBUILD_SRC_DIR/$docker_dir;
  docker build -t $app:$tag .;
  image_id=$(docker images -q $app:$tag);
    for env in $envs; do
    repo=$(cat $CODEBUILD_SRC_DIR/infrastructure/$app-$env.params.json | jq '.Parameters.ContainerImage' | sed 's/"//g');
    region=$(echo $repo | cut -d'.' -f4);
    $(aws ecr get-login --no-include-email --region $region);
    docker tag $image_id $repo;
    docker push $repo;
    done;
  done;
done;

/codebuild/output/tmp/script.sh: line 6: cd: /codebuild/output/src285684034/src/fe/Dockerfile: Not a directory
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /codebuild/output/src285684034/src/Dockerfile: no such file or directory
```

The issue appears to be because the `buildspec` expects the path in the manifest to be a directory where the Dockerfile is present, whereas instead the value in the manifest points to the Dockerfile path: `fe/Dockerfile` instead of `fe`.

**Comment by bvtujo** [comment]
What is the exact contents of the buildspec dockerfile path? I have had app deploy fail when the manifest specifies `Dockerfile`, only to succeed when I change it to `./Dockerfile`.

**Comment by efekarakus** [comment]
in the scenario above it's "fe/Dockerfile", that's why the error is:
`/codebuild/output/tmp/script.sh: line 6: cd: /codebuild/output/src285684034/src/fe/Dockerfile: Not a directory`

**Comment by msafder** [comment]
I've been running into this issue since 0.0.7. I think has to with how it's trying 'cd' to a Dockerfile path that is malformed. Look for this text below: 

src/./SpatialWebInterfaceFlow

```
[Container] 2020/05/05 18:53:44 Running command for app in $apps; do
  for docker_dir in $(cat $CODEBUILD_SRC_DIR/ecs-project/$app/manifest.yml | ruby -ryaml -rjson -e 'puts JSON.pretty_generate(YAML.load(ARGF))' | jq '.image.build' | sed 's/"//g'); do
  cd $CODEBUILD_SRC_DIR/$docker_dir;
  docker build -t $app:$tag .;
  image_id=$(docker images -q $app:$tag);
    for env in $envs; do
    repo=$(cat $CODEBUILD_SRC_DIR/infrastructure/$app-$env.params.json | jq '.Parameters.ContainerImage' | sed 's/"//g');
    region=$(echo $repo | cut -d'.' -f4);
    $(aws ecr get-login --no-include-email --region $region);
    docker tag $image_id $repo;
    docker push $repo;
    done;
  done;
done;
```

```
/codebuild/output/tmp/script.sh: line 6: cd: /codebuild/output/src624709322/src/./SpatialWebInterfaceFlow/GraphDbInt/Dockerfile: Not a directory
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /codebuild/output/src624709322/src/Dockerfile: no such file or directory

```

**Comment by efekarakus** [comment]
@msafder we released the fix in v0.0.9! :) https://github.com/aws/amazon-ecs-cli-v2/releases/tag/v0.0.9
